Alignment/OfflineValidation/python/TkAlAllInOneTool/geometryComparisonTemplates.py

```diff
@@ -50,15 +49,11 @@
 # global tag
 process.load("Configuration.StandardSequences.FrontierConditions_GlobalTag_cff")
 process.GlobalTag.globaltag = ".oO[GlobalTag]Oo." 
-process.load("Configuration.Geometry.GeometryDB_cff")
-
-#process.load("Geometry.TrackerNumberingBuilder.trackerNumberingGeometry_cfi")
 
-#process.load("Geometry.TrackerGeometryBuilder.trackerGeometry_cfi")
+process.load('Configuration.Geometry.GeometryExtended2015_cff')
+process.TrackerTopologyEP = cms.ESProducer("TrackerTopologyEP")
 
-#process.load("Alignment.CommonAlignmentProducer.GlobalPosition_Frontier_cff")
-# the input .GlobalPosition_Frontier_cff is providing the frontier://FrontierProd/CMS_COND_31X_ALIGNMENT in the release which does not provide the ideal geometry
-#process.GlobalPosition.connect = 'frontier://FrontierProd/CMS_COND_31X_FROM21X'
+process.load("Geometry.TrackerGeometryBuilder.trackerGeometry_cfi")i")
```

Comment:
Yes it is a typo, probably I made it when erasing commented links this morning.
Do I need to change the pull request? Unfortunately, I am travelling at the moment and not sure whether our cheap hotel has wifi tonight. 
Can someone of you fix it?